src/blog/_posts/2017-08-15-opera-47.md

```diff
@@ -0,0 +1,74 @@
+* [`PushSubscription.expirationTime`](https://w3c.github.io/push-api/#dom-pushsubscription-expirationtime) is now available, notifying sites when and if a subscription will expire.
+* To improve performance and predictability, [`pointermove`](https://developer.mozilla.org/en-US/docs/Web/Events/pointermove) and [`mousemove`](https://developer.mozilla.org/en-US/docs/Web/Events/mousemove) events are now delivered once per animation frame (analog to `requestAnimationFrame`), matching the current functionality of [scroll](https://developer.mozilla.org/en-US/docs/Web/Events/scroll) and [`TouchEvents`](https://developer.mozilla.org/en-US/docs/Web/API/TouchEvent).
+* The `:focus-within` CSS pseudo-class is now available, [affecting](https://codepen.io/anon/pen/xOLJym) [any](https://blogs.igalia.com/mrego/files/2017/05/focus-within-menu.html) [element](https://blogs.igalia.com/mrego/files/2017/05/focus-within-demo.html) the `:focus` pseudo-class affects, as well as any element with a descendant affected by `:focus`.
+* The CSS [`frames()` timing function](https://www.w3.org/TR/css-timing-1/#frames-timing-functions) is now available, [allowing for proper frame-based animations](https://docs.google.com/document/d/1dxY0VLynyZzPr-KSOf7eIedhC-fjGaR2NTfWMZDjSTc/edit#). (Animations where all frames have exactly the same duration, including its first and last frames.)
+* To provide an improved way to capture editing actions, [`InputEvent`](https://www.w3.org/TR/2017/WD-input-events-1-20170321/) now allows user input to be managed by script, enhancing the details provided to editable elements.
+* To increase security, a `beforeunload` dialog triggered when the user leaves a site will now only be shown if the frame attempting to display it has ever received a user gesture or user interaction, though the [`beforeunload` event](https://html.spec.whatwg.org/multipage/browsers.html#unloading-documents) will still be dispatched regardless.
+* VP9, an open and royalty-free video coding format, can now be [used](https://cs.chromium.org/chromium/src/media/test/data/bear-320x240-v_frag-vp9.mp4) with the [MP4 (ISO BMFF) container](http://www.webmproject.org/vp9/mp4/) and requires the new VP9 string format mentioned below.
+* A new [VP9 string format](http://www.webmproject.org/vp9/mp4/#codecs-parameter-string) is now available and accepted by various [media-related APIs](https://googlechrome.github.io/samples/media/vp9-codec-string.html), enabling developers to describe the encoding properties that are common in video codecs, but are not yet exposed.
+
+## Deprecations and interoperability improvements
+
+* `getElementsByTagName()` now accepts qualified names in response to an update to the [DOM specification](https://dom.spec.whatwg.org/#concept-getelementsbytagname).
+* `/deep/` now behaves like the [descendant combinator](https://drafts.csswg.org/css-scoping/#deep-combinator), which is effectively a no-op.
```

Comment:
I think this could use some more context, like 

```
* The [`/deep/` combinator](https://www.w3.org/TR/2014/WD-css-scoping-1-20140403/#deep-combinator) which has been removed from the [current specification](https://drafts.csswg.org/css-scoping/), now behaves like the [descendant combinator](https://drafts.csswg.org/selectors-4/#descendant-combinator), which is effectively a no-op. Previously it would enable selecting descendants in shadow trees.
```